File: Challenge19.js
function array_front9(nums){
  var result;
  if (nums.length < 4) {
    for (var i = 0; i < nums.length; i++) {
      if (nums[i] == 9) {
        result = true;break;
      }
       else {
         result = false;continue;
       }
    }
  }
  else {
    var num4 = nums.splice(0,4);
    for (var i = 0; i < num4.length; i++) {
      if (num4[i] == 9) {
        result = true;break;
      }
      else {
        result = false;continue;
      }
    }
  }
  return result;
}

File: Challenge19.py
def array_front9(nums):
    #if length of nums is 4 or more
    if len(nums) >= 4:
        #for i in the 4 length of nums
        for i in nums[0:4]:
            #check if each number is 9 or not and then return!
            if i == 9:
                return i == 9
        return i == 9
    else:
        for i in nums:
            if i == 9:
                return i == 9
        return i == 9
print(array_front9([1, 2, 3, 4, 5]))
